src/components/ConnectWallet/ConnectWalletDialog.tsx

```diff
@@ -0,0 +1,229 @@
+import { Trans } from '@lingui/macro'
+import { Web3ReactHooks } from '@web3-react/core'
+import { EIP1193 } from '@web3-react/eip1193'
+import { Url } from '@web3-react/url'
+import { URI_AVAILABLE, WalletConnect } from '@web3-react/walletconnect'
+import METAMASK_ICON_URL from 'assets/images/metamaskIcon.png'
+import WALLETCONNECT_ICON_URL from 'assets/images/walletConnectIcon.svg'
+import Button from 'components/Button'
+import Column from 'components/Column'
+import { Header } from 'components/Dialog'
+import Row from 'components/Row'
+import EventEmitter from 'events'
+import { connections, defaultChainIdAtom, Web3Connection } from 'hooks/connectWeb3/useWeb3React'
+import { atom, useAtom } from 'jotai'
+import { useAtomValue } from 'jotai/utils'
+import QRCode from 'qrcode'
+import { useCallback, useEffect, useState } from 'react'
+import styled from 'styled-components/macro'
+import { lightTheme, ThemedText } from 'theme'
+
+const Body = styled(Column)`
+  height: calc(100% - 2.5em);
+`
+
+const SecondaryOptionsRow = styled(Row)`
+  align-self: end;
+  grid-template-columns: repeat(2, calc(50% - 0.75em / 2));
+  height: fit-content;
+`
+
+const ButtonContents = styled(Column)`
+  gap: 0.75em;
+  justify-items: center;
+`
+
+const StyledMainButton = styled(Button)`
+  background-color: ${({ theme }) => theme.container};
+  border-radius: ${({ theme }) => theme.borderRadius * 0.75}em;
+  height: 183px;
```

Comment:
not a huge deal, but generally if i see numbers not divisible by 4 in css i get slightly concerned. is this trying to align with another part of the page?

Reply:
@kristiehuang can you confirm that this is as spec'd and an intentional % 4 exception from design?